Gems/ROS2/Code/Source/Lidar/LidarTemplateUtils.cpp

```diff
@@ -12,16 +12,17 @@
 
 namespace ROS2
 {
-    LidarTemplate LidarTemplateUtils::GetTemplate(LidarTemplate::LidarModel model)
+    namespace
     {
         using Model = LidarTemplate::LidarModel;
 
-        static const std::unordered_map<Model, LidarTemplate> templates = {
+        static const AZStd::map<Model, LidarTemplate> templates = {
             {
                 Model::Custom3DLidar,
                 {
                     /*.m_model = */ Model::Custom3DLidar,
                     /*.m_name = */ "CustomLidar",
+                    /*.m_is2D = */ false,
```

Comment:
Do we need `m_is2D`? Can't we just deduce that from `m_layers` (`m_layers == 1`)?